### ISSUE TITLE
Dynamic price thresholds

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#945]
Enhancement of https://github.com/ExchangeUnion/xud/issues/163 (https://github.com/ExchangeUnion/xud/pull/921), making price thresholds dynamic (+-X% from current mid market price).

[Things to consider](https://github.com/ExchangeUnion/xud/pull/921#issuecomment-488234241).

**Comment by kilrau** [comment]
Thanks for the feedback to bump prio of this one. Updated the description. Wdyt? @michael1011

**Comment by michael1011** [comment]
I agree with the concept. In my opinion it should be implemented in `xucli` and other applications building on XUD and not XUD itself since the definition of *sane* is subjective and consumers of the XUD API, like arby, probably have own sanity checks in place already and should know better than XUD anyways.

10% might be a little conservative when considering the current liquidity in the network but that is just a constant which can easily be changed once the actual logic is in place.
